cocos/gfx/webgl2/webgl2-commands.ts

```diff
@@ -2774,11 +2788,16 @@ export function WebGL2CmdFuncCopyTexImagesToTexture (
 
     switch (gpuTexture.glTarget) {
     case gl.TEXTURE_2D: {
-        for (let k = 0; k < regions.length; k++) {
-            const region = regions[k];
-            gl.texSubImage2D(gl.TEXTURE_2D, region.texSubres.mipLevel,
-                region.texOffset.x, region.texOffset.y,
-                gpuTexture.glFormat, gpuTexture.glType, texImages[n++]);
+        if (isUseTexImage2D(texImages, regions)) {
```

Comment:
Shall we rename 'isUseTexImage2D' to 'toUseTexImage2D'?

Reply:
yes we can